disco_aws_automation/disco_elastigroup.py

```diff
@@ -434,32 +434,33 @@ def delete_all_recurring_group_actions(self, hostclass=None, group_name=None):
     def create_recurring_group_action(self, recurrance, min_size=None, desired_capacity=None, max_size=None,
                                       hostclass=None, group_name=None):
         """Creates a recurring scheduled action for a hostclass"""
-        existing_group = self.get_existing_group(hostclass=hostclass, group_name=group_name)
-
-        task = {
-            'cronExpression': recurrance,
-            'taskType': 'scale'
-        }
+        existing_groups = self.get_existing_groups(hostclass=hostclass, group_name=group_name)
+        for existing_group in existing_groups:
```

Comment:
If schedule is updated during a blue/green deployment, and new AMI fails to pass the tests, we are making sure that at least the _old_ group that will remain after blue/green gets the _new_ schedule.

👍